src-extras/Database/LSMTree/Extras/Generators.hs

```diff
@@ -46,6 +46,7 @@ import           Data.List (sort)
 import qualified Data.Primitive.ByteArray as BA
 import qualified Data.Vector.Primitive as VP
 import           Data.Word
+import qualified Database.LSMTree as LSMTree
```

Comment:
To fit the pattern of `Normal`/`Monoidal`, would it make sense to import it as `Unified` here and in other places?